Add Intra_R_Add_Exocyclic training reactions from Wang et al. 2015

mliu49 opened the pull request:
Should be pretty straightforward. The Intra_R_Add_Endocyclic reactions were added by me a long time ago. Someone else was supposed to add the Intra_R_Add_Exocyclic reactions, but it never happened. Prior data for these reactions were from rate rules only.

The intra_H_migration reactions from the same paper also do not seem to have been added, but that'll be another PR when I get around to it.

Reference: [Wang, K.; Villano, S. M.; Dean, A. M. J. Phys. Chem. A 2015, 119 (28), 7205–7221.](http://pubs.acs.org/doi/abs/10.1021/jp511017z)

alongd commented on input/kinetics/families/Intra_R_Add_Exocyclic/training/reactions.py:
```diff
@@ -6824,3 +6824,867 @@
+    longDesc =
+u"""
+Quantum chemistry calculations CBS-QB3 calculation with 1-d rotor treatment at 
+B3LYP/631G(d)" using Gaussian 03 and Gaussian 09. High-pressure-limit rate 
```
Could you replace all `B3LYP/631G(d)"` with `B3LYP/6-31G(d)` in this file? While at it, we could also modify the shortDescs to have the dashed version `6-31G(d)`

alongd commented on input/kinetics/families/Intra_R_Add_Exocyclic/training/reactions.py:
```diff
@@ -6824,3 +6824,867 @@
+    index = 335,
+    label = "C4H7 <=> C4H7-2",
+    degeneracy = 1.0,
+    kinetics = Arrhenius(A=(6.32e+08, 's^-1'), n=0.97, Ea=(8.9, 'kcal/mol'), T0=(1, 'K')),
```
Could you add `Tmin = (300, 'K'), Tmax = (2500, 'K')` to all the new training reactions? (see p. 3 of the paper, top-left: "Rate constants were calculated over a temperature range of 300−2500 K in 50 K increments and fit to modified Arrhenius expressions")

alongd commented on input/kinetics/families/Intra_R_Add_Exocyclic/training/reactions.py:
```diff
@@ -6824,3 +6824,867 @@
+u"""
+Quantum chemistry calculations CBS-QB3 calculation with 1-d rotor treatment at 
+B3LYP/631G(d)" using Gaussian 03 and Gaussian 09. High-pressure-limit rate 
+coefficient computed TST with Eckart Tunnelling"
```
Can you replace `coefficient computed TST with Eckart Tunnelling` with `coefficient computed using TST with Eckart Tunnelling` throughout this file?

mliu49 commented:
Thanks for the comments! I fixed the new training reactions and also updated the old ones in Intra_R_Add_Endocyclic.

I also removed the uncertainty on Ea from the Endocyclic training reactions because I don't think +/- 1 kcal is appropriate. I didn't know what I was doing back then...

alongd commented:
Great! Squash and rebase?

mliu49 commented:
Done! This should be good to go.
